### PR TITLE
dbvis: Drop 32bit arch, update 64bit url and hash,

### LINKED ISSUE #9313: dbvis out of date
## Bug Report

dbvis

### Current Behaviour

dbvis version is 13.0.5

### Expected Behaviour

Latest version is Version 14.0 (released on 2022-09-20)

### Additional context/output

Support for 32bit architecture dropped in 14.0. Seems to be causing the autoupdate to fail.

https://www.dbvis.com/releasenotes/

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #9313

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HUMORCE** [comment]
the package without JRE is more suitable for Scoop.

**Comment by Zliced13** [comment]
Ok, it's fixed now.

**Comment by Zliced13** [comment]
Try the manifest from my personal bucket if you want to:
```
scoop bucket add MyScoop 'https://github.com/Zliced13/MyScoop'
```
```
scoop install dbvis_fixed
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]
`suggest` of JRE/JDK is missing.

File: bucket/dbvis.json [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
     "license": {
         "identifier": "Freeware",
         "url": "https://www.dbvis.com/eula/"
     },
     "suggest": {
-        "JDK": [
-            "java/oraclejdk",
+        "JDK/JRE": [
+            "java/temurin-jre",
             "java/openjdk"
```
Comment (HUMORCE) [inline bucket/dbvis.json:12]: separate them or simply drop JDK part.

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/dbvis.json (hunk 1 above)
Comment (Zliced13) [reply bucket/dbvis.json:12]: Ok I will when I can, just currently busy with personal stuff at the moment.

Comment (HUMORCE) [reply bucket/dbvis.json:12]: sure, good day.

Comment (Zliced13) [reply bucket/dbvis.json]: Done.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**